dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.17.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib flag-o-matic
+
+DESCRIPTION="MariaDB Connector/ODBC"
+HOMEPAGE="https://downloads.mariadb.org/connector-odbc/"
+SRC_URI="mirror://mariadb/connector-odbc-${PV}/${P}-src.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/3.1"
+KEYWORDS="amd64 x86"
+IUSE="ssl"
+
+S="${S}-src"
+
+# USE=ssl merely enables the configuration options (seemingly for interactive
+# sessions) and does not cause direct linking to any SSL libraries.  However,
+# it doesn't make sense enable these configuration options unless the
+# underlying mariadb-connector-c has ssl enabled, thus if we have USE=ssl,
+# require mariadb-connector-c to have it too.
+DEPEND="=dev-db/mariadb-connector-c-$(ver_cut 1-2)*:=[ssl?]
+	dev-db/unixODBC"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	cmake_src_prepare
+
+	sed -e "s,/lib/,/$(get_libdir)/,g" "${FILESDIR}/odbcinst2.ini" > odbcinst.ini || die
+}
+
+multilib_src_configure() {
+	append-cppflags $(mariadb_config --cflags || die)
+	local mycmakeargs=(
+		-DWITH_SSL=$(usex ssl OPENSSL OFF)
+		-DMARIADB_LINK_DYNAMIC=YES
+		-DUSE_SYSTEM_INSTALLED_LIB=YES
+		-DINSTALL_DOCDIR="/usr/share/doc/${PF}"
+		-DINSTALL_LICENSEDIR="/usr/share/doc/${PF}"
```

Comment:
Does this actually install license files? Because it shouldn't, instead just cite correct LICENSE and rely on system copy.

Reply:
```
jkroon@plastiekpoot ~ $ equery files mariadb-connector-odbc
 * Searching for mariadb-connector-odbc ...
 * Contents of dev-db/mariadb-connector-odbc-3.1.17:
...
/usr/share/doc/mariadb-connector-odbc-3.1.17/COPYING.bz2
/usr/share/doc/mariadb-connector-odbc-3.1.17/README.bz2
/usr/share/doc/mariadb-connector-odbc-3.1.17/README.md.bz2
```

So yes it does.  Suppose the readme's are fine.  Will filter this.

Reply:
So I couldn't see a way in the cmake system to prevent installation, so instead I just rm'ed it again at a later stage, please confirm this is fine.

Reply:
The only way would be patching CMakeLists.txt.

Reply:
May I assume the rm after install solution is acceptable?